image/cli/mascli/functions/gitops_suite

```diff
@@ -565,6 +558,26 @@ function gitops_suite() {
   fi
   mkdir -p ${GITOPS_INSTANCE_DIR}
 
+  # Set certs only if manual cert is True (to create k8s secret in gitops)
+  # ---------------------------------------------------------------------------
+  
+  if [ "${MAS_MANUAL_CERT_MGMT}" == "True" ] ; then
+    if [[ -n "$MAS_MANUAL_CERTS_YAML" && -s "$MAS_MANUAL_CERTS_YAML" ]]; then
```

Comment:
what does the -s option do here? Should that be -f to make sure the file exists?

Reply:
Added -s (instead of -f) as the logic checks for a file with nonzero size

-s filename - Check if file is nonzero size
-f filename - Check for regular file existence not a directory